src/main.rs

```diff
@@ -1,31 +1,19 @@
-extern crate systemstat;
-use chrono::DateTime;
-use chrono::TimeZone;
-use chrono::Utc;
-use socketcan::*;
-use std::env;
-use std::io::Write;
-use std::os::unix::net::UnixStream;
-use std::process::Command;
-use std::sync::mpsc::channel;
-use std::thread;
-mod data;
-mod decode_data;
-mod master_mapping;
-mod message;
+use std::{
+    env,
+    process::{self, Command},
+};
 
-fn main() {
-    let args: Vec<String> = env::args().collect();
-    let default = "tmp/ipc.sock".to_owned();
-    let ipc_path = args.get(0).unwrap_or(&default);
+use calypso::{client::Client, message::Message, mqtt::MqttClient};
+use socketcan::CANSocket;
 
+fn configure_can() {
     let mut down_command = Command::new("sudo")
```

Comment:
Nit: is there any way we can sidestep using sudo here? Feels weird for an application to be using a sudo command when being deployed when we could do something with user groups or something. Could also be necessary so just food for thought

Reply:
I dont know thatd more of a jack linux question. Hubert made this code i believe